src/main/integrations/net-breadcrumbs.ts

```diff
@@ -202,6 +209,24 @@ function createWrappedRequestFactory(
           if (shouldAttachTraceData(method, url)) {
             request.setHeader('sentry-trace', span.toTraceparent());
```

Comment:
we need to add baggage here as well for this case, I guess we never ended up doing this.

Reply:
oh yes, good spot!